### PR TITLE
Add SideMenu to all pages

### PR DESCRIPTION
**Pull request by huishi329**

### CODE REVIEW

File: react-app/src/History.js [hunk 1]
```diff
@@ -42,7 +47,7 @@ const styles = StyleSheet.create({
   calendar: {
     // to reviewers
```
Comment (mylesieong) [inline react-app/src/History.js:48]: Are these comments need? If not, I suggest we clean them up

File: react-app/src/SideMenu.js [hunk 2]
```diff
@@ -15,10 +21,11 @@ const styles = StyleSheet.create({
...
         position: 'absolute',
         top: 10,
-        zIndex: 1000,
+        left: 0,
+        zIndex: 10,
         backgroundColor: '#D0D7DD',
-        width: 300,
-        minHeight: 600,
+        width: '60%',
+        height: '100%',
```
Comment (mylesieong) [inline react-app/src/SideMenu.js:28]: Can we add elevation here?

File: react-app/src/History.js (hunk 1 above)
Comment (akaka0039) [reply react-app/src/History.js:48]: yes we can clean up
※I just wanted to explain how  "...StyleSheet.absoluteFillObject" works

**Review by j3rryzh — APPROVED** [review]
I've done some code format change

**Pull request closed without merging**